app/src/main/res/layout/activity_user_type_choice.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".UserTypeChoice">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+
+        tools:layout_editor_absoluteX="169dp"
+        tools:layout_editor_absoluteY="295dp">
+
+        <RadioGroup
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentTop="true"
+            android:layout_marginStart="153dp"
+            android:layout_marginTop="307dp">
+
+            <RadioButton
+                android:id="@+id/radioProfessorType"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/professor"></RadioButton>
+
+            <RadioButton
+                android:id="@+id/radioStudentType"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/student"></RadioButton>
+        </RadioGroup>
+
+        <TextView
+            android:id="@+id/textViewUserType"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentTop="true"
+            android:layout_marginStart="131dp"
+            android:layout_marginTop="265dp"
+            android:text="@string/selection"
+            android:textSize="20sp" />
+
+
+    </RelativeLayout>
```

Comment:
The UI looks fine but the code is a little "aproximatif" with absolute positions/etc, it could be better to use the constraint layout such as in the suggestion.
Because if we use different screen size, the elements will not be centered.

![image](https://user-images.githubusercontent.com/19479532/116477654-30eaba80-a87d-11eb-8937-0d39b124cd30.png)

(If you use this suggestion, the tests would need to be modified because of the relative layout removed)
```suggestion
    <RadioGroup
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_alignParentStart="true"
        android:layout_alignParentTop="true"
        android:layout_marginTop="20dp"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintHorizontal_bias="0.5"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toBottomOf="@+id/textViewUserType">

        <RadioButton
            android:id="@+id/radioProfessorType"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="@string/professor" />

        <RadioButton
            android:id="@+id/radioStudentType"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="@string/student" />
    </RadioGroup>

    <TextView
        android:id="@+id/textViewUserType"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="@string/selection"
        android:textSize="20sp"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintHorizontal_bias="0.5"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent"
        app:layout_constraintVertical_bias="0.4" />
```